android-components/components/feature/addons/build.gradle

```diff
@@ -38,9 +38,6 @@ android {
     lint {
         warningsAsErrors true
         abortOnError true
-
-        // https://github.com/mozilla-mobile/android-components/issues/10642
-        disable 'UnusedResources'
```

Comment:
Are we removing these checks because lint will now correctly pass because all the resources are correctly marked with `UnusedResources`?

Reply:
Now lint pass as it's able to see all the resources being used, before when we have resources in different repositories, lint was not aware that they were used.